kolibri/plugins/learn/assets/src/vue/icons/navigate_before.svg

```diff
@@ -0,0 +1,4 @@
+<svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This is also common for "up a level in the hierarchy": ![](http://scanscope2.path.med.umich.edu:82/html/folder_up.gif)